src/ssmDocument/explorer/documentItemNode.ts

```diff
@@ -30,6 +37,17 @@ export class DocumentItemNode extends AWSTreeNodeBase {
         return this.documentItem.Owner || ''
     }
 
+    public async executeDocument() {
+        let executeDocumentUrl: string =
+            'https://console.aws.amazon.com/systems-manager/automation/execute/' +
+            this.documentName +
+            '?region=' +
+            this.regionCode
+        return await vscode.env.openExternal(vscode.Uri.parse(executeDocumentUrl)).then(success => {
+            return success
+        })
```

Comment:
Via offline chat with @awschristou , we _highly_ recommend against deep links to the console. The reasoning behind this is that there's no guarantee that the account in VS Code matches with the account in the console, so it can potentially be a bad experience if the IDE links the user to nowhere. Are there any other ways to programmatically trigger this? It might also be a good idea to instead pop a information box that can copy the URL to the clipboard so the user has the ability to navigate there on their own volition.

Reply:
I'd state it stronger than a recommendation, we can't presently ship deep links to the console because of the broken experience @bryceitoc9 specified, it's something on our roadmap but not currently supported.

Reply:
I'll remove the deep link to console and look for other ways to provide a URL to the users

Reply:
Removed the command completely for now, will work on this idea for future PRs